pom.xml

```diff
@@ -112,6 +112,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
     <commons-collections.version>3.2.2</commons-collections.version>
     <commons-compress.version>1.21</commons-compress.version>
     <commons-configuration2.version>2.1.1</commons-configuration2.version>
+    <zstd-jni.version>1.5.0-2</zstd-jni.version>
```

Comment:
Let's use the latest version 1.5.2-5
https://mvnrepository.com/artifact/com.github.luben/zstd-jni/1.5.2-5

Reply:
finished